### DISCUSSION TITLE
请问hexo如何实现代码演示，就像elementui里演示组件效果那样Could you please tell me how to achieve code demonstrations in Hexo, similar to the component effect demonstrations in ElementUI?

### DISCUSSION THREAD

**Discussion by xiaoyuebao** [hexojs/hexo#5580] [Question / Answer]
请问hexo如何实现代码演示，就像elementui里演示组件效果那样
Could you please tell me how to achieve code demonstrations in Hexo, similar to the component effect demonstrations in ElementUI?
![image](https://github.com/user-attachments/assets/a938db77-ac51-4df1-a602-b3ea66bf3804)

**Comment by stevenjoezhang** [comment]
据我所知没有现成的插件可以实现这个功能，你可以试试基于一些具有演示组件效果功能的前端库进行实现
